modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/processors/SchemaProcessor.java

```diff
@@ -157,8 +157,10 @@ public void processComposedSchema(ComposedSchema composedSchema) {
                     }
                 }
             }
-        }if(composedSchema.getOneOf() != null){
-            final List<Schema> schemas = composedSchema.getOneOf();
+        }
```

Comment:
AS mentioned above:

IIANM the changes here consider `oneOf` and `anyOf` as mutually exclusive, while they can coexist even if not so common. I would suggest to consider them both in the processing

Reply:
Updated as suggested.